src/config.js

```diff
@@ -14,12 +14,33 @@
  *  limitations under the License.
  */
 
-const config = {
-    clientId: 'react-client',
-    redirectUri: 'http://localhost:3000/',
+const url= new URLSearchParams(window.location.search);
+let clientId = 'go_react_client_for_personal_user';
+let environment = 'dev'
+
+if (url.has('client')) {
+    clientId = url.get('client')
+}
+
+if (url.has('env')) {
+    environment = url.get('env');
+}
+
+let config = {
+    clientId: clientId,
     scope: 'openid',
-    authorizationEndpoint: 'https://curity.transfergo.land/oauth/v2/oauth-authorize',
-    tokenEndpoint: 'https://curity.transfergo.land/oauth/v2/oauth-token'
+};
+
+if (environment === 'dev') {
```

Comment:
Just for the sake of sharing thoughts :) I would not add this at all because. With `if` one needs to see if it is possible that `environment` is not set. Just set those values as default and then override them if `environment` is defined.